src/lualib/SourceMapTraceBack.ts

```diff
@@ -9,9 +10,13 @@ function __TS__SourceMapTraceBack(this: void, fileName: string, sourceMap: { [li
         debug.traceback = ((thread, message, level) => {
             let trace: string;
             if (thread === undefined && message === undefined && level === undefined) {
-                trace = globalThis.__TS__originalTraceback();
+                trace = (globalThis.__TS__originalTraceback as (this: void, ...args: any) => string)();
```

Comment:
These duplicate casts seem pretty ugly (actually it's ugly these function are any in the first place), would it work to just add
```ts
declare function __TS__originalTraceback(this: void, thread?: LuaThread, message?: string, level?: number);
```
At the top of the file?

Reply:
Yes, that does work...

For some reason, before, I was trying to do `declare let __Ts__originalTraceback : (//)=>any` and that didn't work, so I did the cast instead.